### DISCUSSION TITLE
Google jobs in Sessions in 1.60.0

### DISCUSSION THREAD

**Discussion by masterwishx** [browserless/chrome#3664] [Q&A] [answered]
Using Docker in Unraid changedetection + browserless
When moved from latest to 1-stable-chrome version 
i found many sessions that i not runned :  

![image](https://github.com/browserless/browserless/assets/28630321/60dbf9ec-1443-4657-92ed-d2d7a1434f9a)

**Comment by joelgriffith** [comment, marked as answer]
These sessions are likely background pages as puppeteer surfaces them in their pages when asking for Targets. Chrome stable also loads more code and other stuff than Chromium, so these could also be built-in extensions as well

**Reply by masterwishx** [reply]
Strange but when changed back to lasted , there was no other jobs back then
